From a98327f7e4246991fb4caaa232d8d75aef6fd7ed Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Fri, 28 Sep 2018 14:36:20 +0300
Subject: [PATCH 4765/5242] MLK-19700-2: arm64: dts: imx8mq-evk: Remove
 LCDIF+RM67191 dtb from Makefile

commit  c691163fee4a4964d3958e669ee3ea831924c6b9 from
https://source.codeaurora.org/external/imx/linux-imx.git

The LCDIF resolution has to be limited to 1280x720 due to system
bandwidth limitation. Since the RM67191 DSI panel can't handle this
resolution, just remove the dtb from Makefile.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/Makefile |    2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index 5baf6eb..0a9f8a3 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -90,8 +90,6 @@ dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += fsl-imx8mq-ddr3l-arm2.dtb \
 				 fsl-imx8mq-evk-pcie1-m2.dtb \
 				 fsl-imx8mq-evk-lcdif-adv7535.dtb \
 				 fsl-imx8mq-evk-lcdif-adv7535-b3.dtb \
-				 fsl-imx8mq-evk-lcdif-rm67191.dtb \
-				 fsl-imx8mq-evk-lcdif-rm67191-b3.dtb \
 				 fsl-imx8mq-evk-mipi-csi2.dtb \
 				 fsl-imx8mq-evk-pdm.dtb \
 				 fsl-imx8mq-evk-dcss-adv7535.dtb \
-- 
1.7.9.5

